src/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/DumperInterface.php

```diff
@@ -34,10 +34,11 @@ public function isProxyCandidate(Definition $definition);
      *
      * @param Definition $definition
      * @param string     $id         service identifier
+     * @param string     $methodName the method name to get the service, will be added to the interface in 4.0.
      *
      * @return string
      */
-    public function getProxyFactoryCode(Definition $definition, $id);
+    public function getProxyFactoryCode(Definition $definition, $id/**, $methodName = null */);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://www.codacy.com/app/git_9/symfony/file/2331537317/issues/source?bid=3207821&fileBranchId=3207862#l41)